packages/components/src/button/src/Button.tsx

```diff
@@ -144,7 +144,7 @@ export function InnerButton(props: InnerButtonProps) {
         text: {
             "aria-hidden": loading,
             className: "o-ui-button-text",
-            size: condensed ? condensedTextSize(sizeValue) : sizeValue
+            size: condensed ? condensedTextSize(sizeValue) : "md"
```

Comment:
So you don't want the button text to accept a responsize size when it's not condensed?

Reply:
It has been decided that the current font size on small buttons is too small. md is the minimum a button text should be only the height changes between a small and medium button now.

Reply:
Got it but it should still allow a responsive syntax.

Reply:
enlighten me on the how to do it :)

Reply:
> > Should a Button and a Tag have the same border color?
> > ![image](https://user-images.githubusercontent.com/794579/153620058-e2430ff0-50f0-473a-ace3-d1fe3d1c7e28.png)
> 
> It's just my POV.
> 
> The whole area of a button is cleavable ≠ The clickable area (for closing) of the tag is just the area of the cross button
> 
> > so the graphic rendering of hovering is different
> 
> As user, i easy identify the differents components side by side. I know that are different and that they haven’t the same function

@Chicoalvi make sense, thanks!

Reply:
> enlighten me on the how to do it :)

You can do it with `useResponsiveValue`. To enforce the the minimum size constraint, you either use either `createSizeAdapter` or `adaptSize`.

DM me if you need help.